compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -2027,6 +2027,13 @@ private void resolveFields(BStructureType structureType, BLangStructureTypeNode
                 .map(field -> new BField(names.fromIdNode(field.name), field.pos, field.symbol))
                 .collect(getFieldCollector());
 
+        List<BType> list = new ArrayList<>();
+        for (BLangType tRef : structureTypeNode.typeRefs) {
+            BType type = tRef.type;
+            list.add(type);
+        }
+        structureType.typeInclusions = list;
+
```

Comment:
Do we also have to persist this in BIR and read for symbols coming from a compiled package?

Reply:
Yeah, good point. Need to persist. Will add it.

Reply:
Added with 2b15df7. Created #27035 to track adding test cases related to this.